vignettes/adrs_gcig.Rmd

```diff
@@ -0,0 +1,799 @@
+
+The function `admiral::derive_extreme_event()` can be used to derive the CA-125 Best Confirmed Overall Response Parameter.
+
+Some events such as `bor_cr`, `bor_pr` have been defined in {admiralonco}. Missing events specific to GCIG criteria are defined below.  
+**Note**: *For `SD`, it is not required as for RECIST1.1 that the response occurs after a protocol-defined number of days.*
+
+```{r}
+ovr <- ovr_ca125
+
+bor_sd_gcig <- event(
+  description = "Define stable disease (SD) for best overall response (BOR)",
+  dataset_name = "ovr",
+  condition = AVALC == "SD",
+  set_values_to = exprs(AVALC = "SD")
+)
+
+bor_ne_gcig <- event(
+  description = "Define not evaluable (NE) for best overall response (BOR)",
+  dataset_name = "ovr",
+  condition = AVALC == "NE",
+  set_values_to = exprs(AVALC = "NE")
+)
+
+adrs <- adrs %>%
+  derive_extreme_event(
+    by_vars = get_admiral_option("subject_keys"),
+    tmp_event_nr_var = event_nr,
+    order = exprs(event_nr, ADT),
+    mode = "first",
+    source_datasets = list(
+      ovr = ovr,
+      adsl = adsl
+    ),
+    events = list(
+      bor_cr, bor_pr, bor_sd_gcig, bor_pd, bor_ne_gcig, no_data_missing
+    ),
+    set_values_to = exprs(
+      PARAMCD = "CBORCA",
+      PARAM = "CA-125 Best Confirmed Overall Response by Investigator",
+      PARAMN = 5,
+      PARCAT1 = "Investigator",
+      PARCAT1N = 1,
+      PARCAT2 = "CA-125",
+      PARCAT2N = 1,
+      AVAL = aval_resp(AVALC),
+      ANL01FL = "Y",
+      ANL02FL = "Y"
+    )
+  )
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
+  adrs,
+  display_vars = exprs(USUBJID, AVISIT, PARAMCD, AVALC, ADT),
+  filter = PARAMCD == "CBORCA" & AVALC != "MISSING"
+)
+```
+
+## Combined Best Unconfirmed Overall Response {#combubor}
+
+For patients who are measurable by both RECIST1.1 and CA-125 the concept of Combined Best Overall Response is used. In our assumptions, RECIST1.1 response is unconfirmed, and the combined response from `RS` domain is based on that unconfirmed RECIST1.1.
+
+In this part of the vignette, we will derive Combined Best Unconfirmed Overall Response based on combined response (`PARAMCD = "OVRR11CA"`) as collected on the CRF.
+
+```{r}
+ovr <- ovr_ubor
+
+adrs <- adrs %>%
+  derive_extreme_event(
+    by_vars = get_admiral_option("subject_keys"),
+    tmp_event_nr_var = event_nr,
+    order = exprs(event_nr, ADT),
+    mode = "first",
+    source_datasets = list(
+      ovr = ovr,
+      adsl = adsl
+    ),
+    events = list(
+      bor_cr, bor_pr, bor_sd_gcig, bor_pd, bor_ne_gcig, no_data_missing
+    ),
+    set_values_to = exprs(
+      PARAMCD = "BORCA11",
+      PARAM = "Combined Best Unconfirmed Overall Response by Investigator",
+      PARAMN = 6,
+      PARCAT1 = "Investigator",
+      PARCAT1N = 1,
+      PARCAT2 = "Combined",
+      PARCAT2N = 3,
+      AVAL = aval_resp(AVALC),
+      ANL01FL = "Y",
+      ANL02FL = "Y"
+    )
+  )
+```
+
+```{r, echo=FALSE}
+dataset_vignette(
```

Comment:
same as above CBORCA

Reply:
I see as above 8 answers and have the correct sorting. Here I have PR assigned because as I mentioned in the discussion with @vinhn23  MOUSEANT variable is only associated with CA-125 answers and not with combined answers. Imo in SUPPRS we should have MOUSEANT variable associated with combined answers as well.

Reply:
@starosto As I look at the CRF, we don't collect the data on mouse antiboldy and any other medical /surg. on combined page, so we can also keep it simple to get the best what is provided in the data for combined assuming the investigator took care of it already.